scripts/sbiActor.js

```diff
@@ -195,7 +195,25 @@ export class sbiActor {
 
                 foundArmorItems = true;
             } else {
-                const item = await sUtils.getItemFromPacksAsync(armorType, "equipment");
+                const getShortArmor = (armorType) => {
+                    switch (armorType) {
+                        case "studded leather":
+                            return "studded leather armor";
+                        case "padded":
+                            return "padded armor";
+                        case "plate":
+                            return "plate armor";
+                        case "half plate":
+                            return "half plate armor";
+                        case "splint":
+                            return "splint armor";
+                        default:
+                            return armorType;
+                    }
+                };
+
+                const armorItem = getShortArmor(armorType);
+                const item = await sUtils.getItemFromPacksAsync(armorItem, "equipment");
```

Comment:
I was wondering if it would be better to check for a null `item` and then rerun the `getItemFromPacksAsync` function by appending " armor" to the name if so. Could be more future proof, unless those are literally the only armor names that have this issue. What do you think?

Reply:
Yep that's a much better idea, and keeps the code more concise, thank you!